bin/terminus

```diff
@@ -76,6 +76,9 @@ if (!is_dir($tokens_dir)) {
     );
 }
 
+// Generate UUID for use as distributed trace id
+$GLOBALS['PANTHEON_TRACE_ID'] = vsprintf('%s%s-%s-%s-%s-%s%s%s', str_split(bin2hex(random_bytes(16)), 4));
```

Comment:
I'm wondering if there is a better place to put this than `$GLOBALS`.

- This is only needed in the Request object, and since that class already has a reference to the container, maybe we could put it in the container.
- Maybe we could put it in a static field of the request object.

I think the best idea, though, would be to make a shared TraceID object in the container, and inject it into every request object. Then, the trace ID could just be an ordinary field of the TraceID object.

Reply:
Agree wrapper script is not the way, but how does container get?  xD